client/src/components/Atoms/ListAndDetail.tsx

```diff
@@ -28,15 +28,17 @@ export const ListAndDetail = styled.div`
   display: grid;
   grid-template-columns: 240px 1fr;
   height: 100%;
-  overflow-y: auto;
   width: 100%;
 `
 
 // ---------- List ----------
 
 const ListContainer = styled.div`
+  display: flex;
+  flex-direction: column;
   border-right: 1px solid ${Colors.LIGHT_GRAY2};
-  overflow-y: auto;
+  height: 0; /* Remove from the height calculation of the parent */
+  min-height: 100%; /* Take up the full height of the parent (based on the detail panel) */
```

Comment:
I found this trick here: https://stackoverflow.com/questions/68216918/dynamic-css-grid-with-max-height-relevent-to-another-columns-content. Gave me a new level of understanding of CSS!